src/BTVNanoCommissioning/helpers/func.py

```diff
@@ -85,7 +107,9 @@ def uproot_writeable(events, include=["events", "run", "luminosityBlock"]):
                     events[bname][n], axis=0
                 ) != len(flatten(events[bname][n])):
                     continue
-
+                # skip IdxG
```

Comment:
Why do we need to skip `IdxG`? I am also not sure what this is referring to.

Reply:
This is the association between jet with a collection of PFCands index. It is a nested list under jet collections which cannot be store with current framework. Indeed there's no association build in PFNano, we can skip it at the moment

Reply:
Doesn't harm to keep that part, even though PFNano has this association not yet